.github/workflows/reusable-phpunit-test.yml

```diff
@@ -95,7 +95,7 @@ jobs:
       mssql:
         image: mcr.microsoft.com/mssql/server:2022-latest
         env:
-          SA_PASSWORD: 1Secure*Password1
+          MSSQL_SA_PASSWORD: 1Secure*Password1
```

Comment:
At least, using `MSSQL_SA_PASSWORD` seems better.

> The SA_PASSWORD environment variable is deprecated. Use MSSQL_SA_PASSWORD instead.
https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver16&tabs=cli&pivots=cs1-bash#run-the-container-2